cell2d.jl

```diff
@@ -373,7 +374,11 @@ function generate_curve()
 
     # Implicit version
     #curve = bajajFit(ints, constraints, 3)
-    curve = ipatchFit(ints, constraints)
+	if use_internal_points
+		curve = ipatchFit(ints, constraints)
+	else
+		curve = ipatchFit(ints, [])
+	end
```

Comment:
Yes, I remember that I have implemented this myself, buy seems like I did not commit. Thanks!